src/components/CardsList/CardsList.js

```diff
@@ -1,17 +1,28 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import Card from '../Card/Card';
 import './CardList.scss';
 import './Search.scss';
+import axios from 'axios';
 import cardsInfo from '../../jsonInfo/CardsInfo';
 
 const CardsList = () => {
   const [cards, setCards] = useState(cardsInfo);
 
   const handleChange = (e) => {
     const inputText = e.currentTarget.value;
-    setCards(cardsInfo.filter((card) => card.name.toLowerCase().includes(inputText.toLowerCase())));
+    axios.get(`http://localhost:3030/projects/${inputText}`)
```

Comment:
What if I am not running server on localhost? Create an *Axios instance* and export it from a file.
You can see an example of good API management here.
https://github.com/IAmVisco/pardon/blob/master/src/api/index.js#L2
Do not blindly copy it though - it has no instances and a bit old.